app/javascript/views/publicChannelPage/SuccessWidget.jsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import { FormattedMessage, injectIntl } from "react-intl";
+import { 
+  SuccessWrapper,
+  SendButton,
+  PaymentButtons,
+  QRLink,
+  SuccessMessageWrapper,
+  SuccessThank,
+  SuccessAmount,
+} from "./PublicChannelPageStyle.js";
+
+class SuccessWidget extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      setStateToStart: props.setStateToStart,
+      amount: props.amount,
+      chain: props.chain,
+    }
+  }
+
+  shareSupport(){
+    // ????
```

Comment:
Curious what this is?

Reply:
We don't have behavior described for the sharesupport button yet.  That's a placeholder for a future ticket where we'll add some behavior in.